tests/migration/ember-addon/steps/augment-options/error-handling-package-json-is-not-a-valid-json.test.js

```diff
@@ -10,8 +10,8 @@ test('migration | ember-addon | steps | augment-options > error handling (corrup
   };
 
   const inputProject = {
-    'package.json': '{\n  name:}',
-    'yarn.lock': 'some code for yarn.lock',
+    'package.json': '{\n  "name": }',
```

Comment:
I meant to test what would happen when the value is missing.

Instead, I had incorrectly tested the case of a JSON key having an incorrect format (because the double quotations are missing).